app/templates/course.html

```diff
@@ -20,6 +20,8 @@ <h2>{{currentProgram.name}}: {{curTermName.name}}</h2>
                     <br>
                     {% if (can_edit == True and curTermName.term_state.order == 1) or current_user.isAdmin == True %}
                         {% include "snips/courseElements/createCourse.html" %}
+                    {% elif (can_edit == False and curTermName.term_state.order == 1) %}
```

Comment:
I thought we changed this to use a name instead of 'order'. That would be much clearer.